control-plane/pkg/reconciler/broker/namespaced_broker.go

```diff
@@ -433,6 +509,22 @@ func unstructuredFromObject(obj runtime.Object) (unstructured.Unstructured, erro
 	return unstructured.Unstructured{Object: unstr}, nil
 }
 
+func appendNamespaceAsOwnerRefsToClusterScoped(namespace *corev1.Namespace) mf.Transformer {
+	return func(resource *unstructured.Unstructured) error {
+		if !isClusterScoped(resource.GetKind()) {
+			return nil
+		}
+
+		ref := *metav1.NewControllerRef(namespace, corev1.SchemeGroupVersion.WithKind("Namespace"))
+		ref.Controller = pointer.Bool(true)
```

Comment:
Aren't we getting that error because we're setting it to be controller? Can we have namespace being non-controller and still be a useful addition?

Reply:
I doubt it but I will try.

Reply:
Sorry, I misread the error message. It is about trying to set ownerRef on a namespace.